packages/ui/src/components/atoms/Icon/styles.scss

```diff
@@ -0,0 +1,9 @@
+[data-fs-icon] {
```

Comment:
niceee

Reply:
Can you please update Icon documentation, with this style import?

(Or should we leave it as default on `layout.scss` - or anywhere else?)

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/3356699/205692813-c6c07f4c-d79b-4eab-9bdf-af9c8bf6bf15.png">